build.rs

```diff
@@ -199,3 +211,67 @@ fn main() {
     import_tflite_types(&tflite_src_dir);
     build_inline_cpp(&tflite_src_dir);
 }
+
+//--- /dev/null
+//+++ b/tensorflow/contrib/lite/tools/make/build_armv8.sh
+//@@ -0,0 +1,22 @@
+//+#!/bin/bash -x
+//+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+//+#
+//+# Licensed under the Apache License, Version 2.0 (the "License");
+//+# you may not use this file except in compliance with the License.
+//+# You may obtain a copy of the License at
+//+#
+//+#     http://www.apache.org/licenses/LICENSE-2.0
+//+#
+//+# Unless required by applicable law or agreed to in writing, software
+//+# distributed under the License is distributed on an "AS IS" BASIS,
+//+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//+# See the License for the specific language governing permissions and
+//+# limitations under the License.
+//+# ==============================================================================
+//+
+//+set -e
+//+
+//+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+//+cd "$SCRIPT_DIR/../../../../.."
+//+
+//+CC_PREFIX=aarch64-linux-gnu- make -j 3 -f tensorflow/contrib/lite/tools/make/Makefile TARGET=armv8 TARGET_ARCH=armv8l SHELL=/bin/bash
+//diff --git a/tensorflow/contrib/lite/tools/make/targets/armv8_makefile.inc b/tensorflow/contrib/lite/tools/make/targets/armv8_makefile.inc
+//new file mode 100644
+//index 0000000000..1ec6628074
+//--- /dev/null
+//+++ b/tensorflow/contrib/lite/tools/make/targets/armv8_makefile.inc
+//@@ -0,0 +1,32 @@
+//+# Settings for armv8
+//+ifeq ($(TARGET),armv8)
+//+  TARGET_ARCH := armv8l
+//+  TARGET_TOOLCHAIN_PREFIX := aarch64-linux-gnu-
+//+
+//+  ifeq ($(TARGET_ARCH), armv8l)
+//+    CXXFLAGS += \
+//+                       -march=armv8-a \
+//+      -funsafe-math-optimizations \
+//+      -ftree-vectorize \
+//+      -fPIC
+//+
+//+    CCFLAGS += \
+//+      -march=armv8-a \
+//+      -funsafe-math-optimizations \
+//+      -ftree-vectorize \
+//+      -fPIC
+//+
+//+    LDFLAGS := \
+//+      -Wl,--no-export-dynamic \
+//+      -Wl,--exclude-libs,ALL \
+//+      -Wl,--gc-sections \
+//+      -Wl,--as-needed
+//+  endif
+//+
+//+  LIBS := \
+//+    -lstdc++ \
+//+    -lpthread \
+//+    -lm \
+//+    -ldl
+//+
+//+endif
```

Comment:
could you explain what this is for?

Reply:
I was using a known good implementation from another tensorflow branch to get the build script working. I've removed the comments and added some extra comments to make it clear why the changes are necessary.